2023-05-18 14:54:31 [main] INFO  me.lab6.server.UDPServer - Server started at MSI/172.30.43.124:26566
2023-05-18 14:54:35 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:54:39 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:54:43 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:54:47 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:54:51 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:54:55 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:54:59 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:55:03 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:55:07 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:55:11 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:55:15 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:55:19 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:55:23 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:55:27 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:55:31 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:55:35 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:55:39 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:55:43 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:55:47 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:55:57 [main] INFO  me.lab6.server.UDPServer - Server started at MSI/172.30.43.124:26566
2023-05-18 14:56:01 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:56:05 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:56:09 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:56:13 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:56:17 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:56:21 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:56:25 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:56:29 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:56:33 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:56:37 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:56:41 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:56:45 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:56:49 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:56:53 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:56:57 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:57:01 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:57:05 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:57:09 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:57:13 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:57:17 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:57:21 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:57:25 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:57:29 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:57:33 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:57:37 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:57:41 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:57:45 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:57:49 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:57:53 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:57:57 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:58:01 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:58:05 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:58:09 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:58:14 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:58:18 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:58:22 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:58:26 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:58:30 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:58:34 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:58:38 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:58:42 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:58:46 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:58:50 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:58:54 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:58:58 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:59:02 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:59:06 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:59:10 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:59:14 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:59:18 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:59:22 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:59:26 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:59:30 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:59:34 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:59:38 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:59:42 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:59:46 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:59:50 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:59:54 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 14:59:58 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:00:02 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:00:06 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:00:10 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:00:14 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:00:18 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:00:22 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:00:26 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:00:30 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:00:34 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:00:38 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:00:42 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:00:46 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:00:50 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:00:54 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:00:58 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:01:02 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:01:06 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:01:10 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:01:14 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:01:18 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:01:22 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:01:26 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:01:30 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:01:34 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:01:38 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:01:42 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:01:46 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:01:50 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:01:54 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:01:58 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:02:02 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:02:06 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:02:10 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:02:14 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:02:18 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:02:22 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:02:26 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:02:30 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:02:34 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:02:38 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:02:42 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:02:46 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:02:50 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:02:54 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:02:58 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:03:02 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:03:06 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:03:10 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:03:14 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:03:18 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:03:22 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:03:26 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:03:30 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:05:35 [main] INFO  me.lab6.server.UDPServer - Server started at MSI/172.30.43.124:26566
2023-05-18 15:05:39 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:05:43 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
2023-05-18 15:05:47 [main] ERROR me.lab6.server.UDPServer - There was an error receiving data: java.net.SocketTimeoutException: Receive timed out
